### PR TITLE
TP-1184 Hidden field support

### PR DESCRIPTION
**Pull request by aktfrikshun**
TP-1184 Hidden field support

### CODE REVIEW

File: app/views/components/forms/question_types/_hidden_field.html.erb [hunk 1]
```diff
@@ -1 +1,4 @@
-<%= hidden_field_tag question.answer_field.to_sym, question.text %>
+<div class="radios" role="group" aria-labelledby="<%= question.answer_field %>">
```
Comment (ryanwoldatwork) [inline app/views/components/forms/question_types/_hidden_field.html.erb:1]: Is the div.radios necessary here?

I think the hidden field should be sufficient.

Comment (aktfrikshun) [reply app/views/components/forms/question_types/_hidden_field.html.erb:1]: Not necessary.  I just like the way the rendered HTML flows with each field sitting in the same type of container.  But I'm cool to strip them out if desired.

Comment (ryanwoldatwork) [reply app/views/components/forms/question_types/_hidden_field.html.erb:1]: Ah, okay to leave the div then.

but, perhaps `class` and `role` could be removed.

Comment (aktfrikshun) [reply app/views/components/forms/question_types/_hidden_field.html.erb]: html display attributes removed

**Review by ryanwoldatwork — APPROVED** [review]

**Pull request merged**